transports/msmq/index.md

```diff
@@ -71,12 +71,14 @@ Although MSMQ has the concept of both [Public and Private queues](https://techne
 
 ## Permissions
 
-| Group          | Permissions          |
-|----------------|----------------------|
-| Owning account | Write, Receive, Peek |
-| Administrators | Full                 |
-| Anonymous      | Write                |
-| Everyone       | Write                |
+| Group          | Permissions         | Description                       |
+|----------------|---------------------|-----------------------------------|
+| Owning account | Send, Receive, Peek | Set by NServiceBus                |
+| Administrators | Full                | Set by NServiceBus                |
+| Anonymous      | Send                | Set by NServiceBus prior to 6.1.0 |
```

Comment:
@ramonsmits i dont think @indualagarsamy is questioning when this happened. just that "prior to 6.1.0 " is ambiguous and not in keeping with our version language through doc.

I think the test should not be 

>  Set by NServiceBus prior to 6.1.0

and instead should be

>  Set by NServiceBus Versions 6.0.x and below